app.js

```diff
@@ -87,7 +87,6 @@ app.get(
   data.acceptedCategoriesByFieldGet
 );
 app.post('/data/acceptedcategories/organization/:id', data.acceptedCategoriesOrganizationPost);
-app.get('/data/categories', data.categoriesGet);
```

Comment:
We would still want to keep this endpoint here though because the Dashboard page also uses it when editing